packages/klevu-ui/src/components/klevu-color-swatch/klevu-color-swatch.css

```diff
@@ -1,14 +1,24 @@
 :host {
   display: block;
+  --klevu-color-swatch-border: #8c8c8c;
+  --klevu-color-swatch-default-background: #000;
+  --klevu-color-swatch-size: 25px;
+  --klevu-color-swatch-selected-color: #2b4af7;
+  --klevu-color-swatch-border-radius: 50%;
 }
 
 .circle {
   margin: 3px;
-  height: 25px;
-  width: 25px;
-  background-color: black;
-  border: 1px solid #8c8c8c;
-  border-radius: 50%;
+  height: var(--klevu-color-swatch-size);
+  width: var(--klevu-color-swatch-size);
+  background-color: var(--klevu-color-swatch-default-background);
+  border: 1px solid var(--klevu-color-swatch-border);
+  border-radius: var(--klevu-color-swatch-border-radius);
   background-size: cover;
   display: inline-block;
 }
+
+.selected {
+  box-shadow: 0px 0px 0px 2px var(--klevu-color-swatch-selected-color);
```

Comment:
<img width="620" alt="Näyttökuva 2023-8-22 kello 13 29 32" src="https://github.com/klevultd/frontend-sdk/assets/1254634/4c8c9bf1-1ad4-4029-bbd8-da1f3e9f73ec">

Looking at the design the selection has same kind of outline border as `klevu-button` focus state has. 

It's done like this:

```css
button:focus:after {
  content: "";
  position: absolute;
  top: -3px;
  left: -3px;
  width: calc(100% + 2px);
  height: calc(100% + 2px);
  border: 2px solid var(--klevu-color-primary);
  border-radius: calc(var(--klevu-border-radius-m) + 2px);
}
```

Reply:
Actually. Looking at the design it looks like there is no border color by default. It's set only if there is a need. Maybe should add a `borderColor` to customization object?

Reply:
I applied the above CSS to my selector. you can check now.

Reply:
Moved border to customization object as discussed